src/languages/clojure.ts

```diff
@@ -0,0 +1,17 @@
+import { createPatternMatchers } from "../util/nodeMatchers";
+import { ScopeType, NodeMatcherAlternative } from "../typings/Types";
+
+const nodeMatchers: Partial<Record<ScopeType, NodeMatcherAlternative>> = {
+  comment: "comment",
+  map: "map_lit",
+
+  // A list is either a vector literal or a quoted list literal
+  list: ["vec_lit", "quoting_lit.list_lit"],
+
+  string: "str_lit",
+
+  // A function call is a list literal which is not quoted
+  functionCall: "~quoting_lit.list_lit!",
+};
```

Comment:
@sdorminey @somanythings Here's an initial set of supported scope types.  I couldn't figure out what to do with keys / values, because they seem to be treated identically by the parse tree 🤔.  I didn't add other things as well because I think it's better to have someone who really knows the language involved in that step

Reply:
yeah it's entirely based on the number of items in the map it needs to be even.
a slight bugbear of mine in keyboard editing as well that I've not figured out. :shrug: the calva, will end up slurping more than I want too

Reply:
Ok cool.  We can solve this one fairly easily in cursorless

Reply:
Ok so fairly easily turned out to be wrong 😅.  But it works now.  Can now say "key", "value", and "item"